stub/src/main/java/io/grpc/stub/ServerCallStreamObserver.java

```diff
@@ -53,7 +53,9 @@
    * service returns its {@code StreamObserver}.
    *
    * <p>Setting the onCancelHandler will suppress the on-cancel exception thrown by
```

Comment:
Has the javadoc mentioned that `onNext()` would throw if the call is cancelled?  It seems only the particular library implementation `ServerCalls.ServerCallStreamObserverImpl.onNext()` does so.

Reply:
No, it hasn't, and can't really be represented in `StreamObserver`'s API. The best we've done is prevent it from throwing for unary responses.